### ISSUE TITLE
Linux [Feature request]

### ISSUE THREAD

**Issue by VPaulV** [syanth/gifcopier#10]
Hi! Would it be possible to add linux support?

**Comment by UsernamePlusPlus** [comment]
> Hi! Would it be possible to add linux support?

Yes. From what I saw, he uses OS specific headers only in cpp files, they are small and he uses wide characters to make commands, which will be executed in shell, I believe. Linux supports wide characters without "w" in names of functions and I think it will be simpler to put commands onto shell on Linux

+ He uses curl and CMake, which are cross-platform, like a lot of code in his program. So it would be definetely possible
